packages/fsengagement/src/inboxblocks/TabbedStoryItem.tsx

```diff
@@ -116,6 +153,16 @@ export default class TabbedStoryItem extends Component<TabbedStoryItemProps, Tab
               height: '100%'
             }}
           />
+          {item.textOverlay && item.textOverlay.enabled && <View
+              style={[textContainerStyle, StyleSheet.absoluteFillObject]}
+          >
+            {!!item.textOverlay && <TextBlock
```

Comment:
is this what you intended?  since you already have `item.textOverlay && item.textOverlay.enabled` on the parent this isn't actually doing anything.  Should probably be `{!!item.textOverlay.Title.text`